Cargo.lock

```diff
@@ -1798,33 +1807,22 @@ dependencies = [
 name = "sev_attestation"
 version = "0.1.0"
 dependencies = [
- "sallyport",
+ "sallyport 0.3.0",
```

Comment:
Looks like sev_attestation still needs updating.

Reply:
I don't see this in my latest push.